dotCMS/src/main/java/com/dotcms/graphql/DotGraphQLContext.java

```diff
@@ -1,23 +1,117 @@
 package com.dotcms.graphql;
 
 import com.liferay.portal.model.User;
+import graphql.kickstart.execution.context.DefaultGraphQLContext;
+import graphql.kickstart.servlet.context.GraphQLServletContext;
+import org.dataloader.DataLoaderRegistry;
 
+import javax.security.auth.Subject;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
-import graphql.servlet.GraphQLContext;
+public class DotGraphQLContext extends DefaultGraphQLContext implements
+        GraphQLServletContext {
 
-public class DotGraphQLContext extends GraphQLContext {
+    private final HttpServletRequest httpServletRequest;
+    private final HttpServletResponse httpServletResponse;
+    private User user;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'user' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5685344)